src/app/App.js

```diff
@@ -18,6 +22,27 @@ class App {
     this.picture = new Picture('picture');
     this.whiteButton = new WhiteButton('whiteButton', 'Hall of fame', 'whiteButton');
     this.whiteButton.addIcon('../../static/assets/ui/hof.svg');
+
+    this.whiteBtn = document.querySelector('.whiteButton');
+    this.whiteBtn.addEventListener('click', () => {
+      const whiteBtnContent = document.querySelector('.whiteButton span');
+      const whiteBtnIcon = document.querySelector('.whiteButton img');
+      const rulesElement = document.querySelector('#rules');
+      const rankingElement = document.querySelector('#ranking-box');
+
+      if (whiteBtnContent.textContent === "Hall of fame") {
```

Comment:
Przenieśmy to do constants: "Hall of fame"

Reply:
gotowe